docs/topics/class-based-views/mixins.txt

```diff
@@ -324,11 +324,6 @@ Now we can write a new ``PublisherDetail``::
             self.object = self.get_object(queryset=Publisher.objects.all())
             return super().get(request, *args, **kwargs)
 
-        def get_context_data(self, **kwargs):
-            context = super().get_context_data(**kwargs)
-            context['publisher'] = self.object
-            return context
-
```

Comment:
Please revert this change, you should remove `get_context_data()` only from the first example in the _"Using FormMixin with DetailView"_ paragraph.

Reply:
Unfortunately, I had a lot of mistakes on the first try,,,

I think, it's ready now.